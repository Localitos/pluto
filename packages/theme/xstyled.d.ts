import "styled-components";
import "@xstyled/system";
import { theme } from "./src/theme/default";

type AppTheme = typeof theme;

declare module "@xstyled/system" {
  // XStyled wants an interace here.
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface Theme extends AppTheme {}
}

declare module "styled-components" {
  // XStyled wants an interace here.
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface DefaultTheme extends AppTheme {}
}
