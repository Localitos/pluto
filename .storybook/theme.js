import { create } from "@storybook/theming";

export default create({
  base: "light",

  // Typography
  fontBase: '"Noto Sans", sans-serif',
  fontSize: "1rem",
});
