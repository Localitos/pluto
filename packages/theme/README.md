# Pluto Theme - Localyze Design System

Pluto is a design system used to build accessible, consistent, and high quality public-facing experiences at Localyze.

The theme library includes the design tokens and themes needed for development.
