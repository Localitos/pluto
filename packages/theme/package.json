{
  "name": "@localyze-pluto/theme",
  "version": "32.1.0",
  "description": "Pluto Design System Theme ",
  "main": "./dist/index.js",
  "module": "./dist/index.mjs",
  "types": "./dist/index.d.ts",
  "sideEffects": false,
  "license": "MIT",
  "files": [
    "dist/**"
  ],
  "publishConfig": {
    "access": "public"
  },
  "repository": {
    "type": "git",
    "url": "git@github.com:Localitos/pluto.git",
    "directory": "packages/theme"
  },
  "scripts": {
    "build": "tsup src/index.ts --format esm,cjs --dts --external react --minify",
    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
    "dev": "tsup src/index.ts --format esm,cjs --watch --dts --external react",
    "lint": "TIMING=1 eslint src/**/*.ts* --fix",
    "prettier-check": "prettier --check package.json tsconfig.json \"./src/**/*.{tsx,ts,json}\"",
    "test": "jest",
    "tsc": "tsc"
  },
  "peerDependencies": {
    "@localyze-pluto/design-tokens": "^1.23.0",
    "@xstyled/styled-components": "^3.8.0",
    "react": "^17.0.2 || ^18.2.0",
    "react-dom": "^17.0.2 || ^18.2.0",
    "styled-components": "^5.3.11"
  },
  "devDependencies": {
    "@localyze-pluto/design-tokens": "^1.23.0",
    "@localyze-pluto/eslint-config": "^1.18.0",
    "@localyze-pluto/tsconfig": "^1.1.0",
    "@testing-library/jest-dom": "^6.4.5",
    "@testing-library/react": "^15.0.7",
    "@testing-library/user-event": "^14.5.2",
    "@types/jest": "^29.5.12",
    "@types/react": "^18.3.3",
    "@types/react-dom": "^18.3.0",
    "@types/styled-components": "^5.1.26",
    "@xstyled/styled-components": "^3.8.0",
    "eslint": "^8.57.0",
    "jest": "^29.7.0",
    "jest-dom": "^4.0.0",
    "jest-environment-jsdom": "^29.7.0",
    "jest-watch-typeahead": "^2.2.2",
    "prettier": "^3.2.5",
    "pretty-quick": "^4.0.0",
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "styled-components": "^6.1.11",
    "ts-jest": "^29.1.4",
    "tslib": "^2.6.2",
    "tsup": "^8.0.2",
    "typescript": "^5.5.3"
  }
}
