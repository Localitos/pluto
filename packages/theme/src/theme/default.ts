import { defaultTheme } from "@xstyled/styled-components";

export const theme = {
  ...defaultTheme,
  borderStyles: {
    borderSolid: "solid",
  },
  borderWidths: {
    borderWidth0: "0px",
    borderWidth10: "1px",
    borderWidth20: "2px",
  },
  colors: {
    colorTextStrongest: "#0F172A",
    colorTextStronger: "#334155",
    colorText: "#64748B",
    colorTextInfo: "#0B1F9C",
    colorTextError: "#B91C1C",
    colorTextWarning: "#B45309",
    colorTextSuccess: "#047857",
    colorTextInverse: "#FFFFFF",
    colorTextLink: "#102EE9",
    colorTextLinkStrong: "#0B1F9C",
    colorTextLinkVisited: "#950FFF",
    colorTextHeadingStronger: "#041162",
    colorTextHeadingStrong: "#102EE9",
    colorTextHeading: "#334155",
    colorBackground: "#FFFFFF",
    colorBackgroundWeakest: "#F8FAFC",
    colorBackgroundWeaker: "#F7F9FF",
    colorBackgroundWeak: "#F1F5F9",
    colorBackgroundStrong: "#CBD5E1",
    colorBackgroundPrimaryWeak: "#4A63FC",
    colorBackgroundPrimary: "#102EE9",
    colorBackgroundPrimaryStrong: "#0B1F9C",
    colorBackgroundPrimaryStrongest: "#041162",
    colorBackgroundDestructive: "#DC2626",
    colorBackgroundDestructiveStrong: "#B91C1C",
    colorBackgroundInfo: "#EBEDFF",
    colorBackgroundSuccess: "#ECFDF5",
    colorBackgroundWarning: "#FFFBEB",
    colorBackgroundError: "#FEF2F2",
    colorBackgroundBody: "#F7F9FF",
    colorBackgroundTodo:
      "linear-gradient(360deg, rgba(16, 37, 233, 0.08) 0%, rgba(36, 63, 235, 0.0733333) 49.48%, rgba(255, 255, 255, 0) 100%)",
    colorBackgroundComplete:
      "linear-gradient(360deg, rgba(82, 244, 174, 0.12) 0%, rgba(131, 247, 197, 0.0575) 57.81%, rgba(255, 255, 255, 0) 100%)",
    colorBackgroundPreview:
      "linear-gradient(360deg, rgba(71, 94, 105, 0.08) 6.19%, rgba(71, 94, 105, 0.0504167) 57.3%, rgba(255, 255, 255, 0) 93.81%)",
    colorBorderWeakest: "#E2E8F0",
    colorBorderWeaker: "#CBD5E1",
    colorBorder: "#64748B",
    colorBorderStrongest: "#0F172A",
    colorBorderError: "#EF4444",
    colorBorderWarning: "#D97706",
    colorBorderSuccess: "#059669",
    colorBorderPrimary: "#102EE9",
    colorIconError: "#DC2626",
    colorIconWarning: "#D97706",
    colorIconSuccess: "#059669",
    colorIconInfo: "#102EE9",
    colorIconWeaker: "#94A3B8",
    colorIconWeak: "#64748B",
    colorIconStrong: "#334155",
    colorIconStronger: "#0F172A",
    colorAvatarInitials: "#4A63FC",
  },
  fontSizes: {
    fontSize10: "0.75rem", // 12px
    fontSize20: "0.875rem", // 14px
    fontSize30: "1rem", // 16px
    fontSize40: "1.25rem", // 20px
    fontSize50: "1.5rem", // 24px
    fontSize60: "2rem", // 32px
    fontSize70: "2.5rem", // 40px
    fontSize80: "3rem", // 48px
    fontSize90: "3.375rem", // 54px
  },
  fontWeights: {
    fontWeightLight: "300",
    fontWeightRegular: "400",
    fontWeightMedium: "500",
    fontWeightBold: "700",
  },
  fonts: {
    fontFamilyModerat: '"Moderat", "Inter", sans-serif',
  },
  lineHeights: {
    lineHeight10: "1rem", // 16px
    lineHeight20: "1.125rem", // 18px
    lineHeight30: "1.25rem", // 20px
    lineHeight40: "1.5rem", // 24px
    lineHeight50: "1.75rem", // 28px
    lineHeight60: "2rem", // 32px
    lineHeight70: "2.5rem", // 40px
    lineHeight80: "3rem", // 48px
    lineHeight90: "3.375rem", // 54px
    lineHeight100: "3.75rem", // 60px
  },
  radii: {
    borderRadius10: "4px",
    borderRadius20: "6px",
    borderRadius30: "8px",
    borderRadius40: "16px",
    borderRadius50: "24px",
    borderRadiusCircle: "50%",
    borderRadiusPill: "100px",
  },
  shadows: {
    shadow: "0px 3px 24px rgba(15, 23, 42, 0.05)",
    shadowStrong: "0px 8px 24px rgba(15, 23, 42, 0.08)",
    shadowFocus:
      "0px 1px 2px rgba(0, 0, 0, 0.05), 0px 0px 0px 2px #FFFFFF, 0px 0px 0px 4px #102EE9",
  },
  sizes: {
    sizeIcon10: "0.75rem", // 12px
    sizeIcon20: "1rem", // 16px
    sizeIcon30: "1.25rem", // 20px
    sizeIcon40: "1.5rem", // 24px
  },
  space: {
    space0: "0rem", // 0px
    space10: "0.125rem", // 2px
    space20: "0.25rem", // 4px
    space30: "0.5rem", // 8px
    space40: "0.75rem", // 12px
    space50: "1rem", // 16px
    space60: "1.25rem", // 20px
    space70: "1.5rem", // 24px
    space80: "1.75rem", // 28px
    space90: "2rem", // 32px
    space100: "2.25rem", // 36px
    space110: "2.5rem", // 40px
    space120: "3rem", // 48px
    space130: "3.25rem", // 52px
    space140: "3.5rem", // 56px
    space150: "3.75rem", // 60px
    space160: "4.5rem", // 72px
    space170: "5rem", // 80px
    space180: "6rem", // 96px
    space190: "6.25rem", // 100px
    spaceNegative10: "-0.125rem", // -2px
    spaceNegative20: "-0.25rem", // -4px
    spaceNegative30: "-0.5rem", // -8px
    spaceNegative40: "-0.75rem", // -12px
    spaceNegative50: "-1rem", // -16px
    spaceNegative60: "-1.25rem", // -20px
    spaceNegative70: "-1.5rem", // -24px
    spaceNegative80: "-1.75rem", // -28px
    spaceNegative90: "-2rem", // -32px
    spaceNegative100: "-2.25rem", // -36px
    spaceNegative110: "-2.5rem", // -40px
    spaceNegative120: "-3rem", // -48px
    spaceNegative130: "-3.25rem", // -52px
    spaceNegative140: "-3.5rem", // -56px
    spaceNegative150: "-3.75rem", // -60px
    spaceNegative160: "-4.5rem", // -72px
    spaceNegative170: "-5rem", // -80px
    spaceNegative180: "-6rem", // -96px
    spaceNegative190: "-6.25rem", // -100px
  },
  states: {
    ...defaultTheme.states,
    loading: '&[aria-busy="true"]',
  },
  zIndices: {
    zIndex0: 0,
    zIndex10: 10,
    zIndex20: 20,
    zIndex30: 30,
    zIndex40: 40,
    zIndex50: 50,
    zIndex60: 60,
    zIndex70: 70,
    zIndex80: 80,
    zIndex90: 90,
  },
};
