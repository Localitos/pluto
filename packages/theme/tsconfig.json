{
  "extends": "@localyze-pluto/tsconfig",
  "compilerOptions": {
    "outDir": "dist/"
  },
  "include": ["**/*.ts", "**/*.tsx", "xstyled.d.ts"],
  "exclude": ["dist", "build", "node_modules"]
}
