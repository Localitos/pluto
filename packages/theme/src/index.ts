import "styled-components";
import "@xstyled/system";
import { theme } from "./theme/default";

type AppTheme = typeof theme;

declare module "@xstyled/system" {
  // XStyled wants an interace here.
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface Theme extends AppTheme {}
}

declare module "styled-components" {
  // XStyled wants an interace here.
  // eslint-disable-next-line @typescript-eslint/no-empty-interface
  export interface DefaultTheme extends AppTheme {}
}

export * from "@xstyled/styled-components";
export * from "./theme/default";
export * from "./styles/base";
export * from "./styles/global";
