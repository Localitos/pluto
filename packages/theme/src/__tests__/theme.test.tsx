import { theme } from "../theme/default";

describe("Theme", () => {
  it("matches the default theme", () => {
    const defaultTheme = theme;
    expect(defaultTheme).toMatchSnapshot();
  });
});
