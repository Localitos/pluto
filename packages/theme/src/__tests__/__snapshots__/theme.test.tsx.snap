// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme matches the default theme 1`] = `
Object {
  "animations": Object {
    "bounce": "x-bounce 1s infinite",
    "ping": "x-ping 1s cubic-bezier(0, 0, 0.2, 1) infinite",
    "pulse": "x-pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite",
    "spin": "x-spin 1s linear infinite",
  },
  "borderStyles": Object {
    "borderStyleDashed": "dashed",
    "borderStyleSolid": "solid",
    "none": "none",
  },
  "borderWidths": Object {
    "borderWidth0": "0px",
    "borderWidth10": "1px",
    "borderWidth20": "2px",
    "borderWidth30": "3px",
    "borderWidth60": "6px",
  },
  "borders": Object {
    "default": "1px solid transparent",
  },
  "colors": Object {
    "bgBodyMain": "#F7F9FF",
    "bgBodyTalent": "#27313F",
    "bgError": "#FEF2F2",
    "bgInfo": "#EBEDFF",
    "bgPrimary": "#FFFFFF",
    "bgSecondary": "#F8FAFC",
    "bgSuccess": "#ECFDF5",
    "bgWarning": "#FFFBEB",
    "borderError": "#B91C1C",
    "borderInfo": "#102EE9",
    "borderPrimary": "#6C7684",
    "borderSecondary": "#D2D7DF",
    "borderSuccess": "#047857",
    "borderWarning": "#B05C09",
    "colorAvatarBackgroundBlue": "#B8C2FF",
    "colorAvatarBackgroundGreen": "#8AFECC",
    "colorAvatarBackgroundLightBlue": "#A9EBE8",
    "colorAvatarBackgroundOrange": "#FBBF24",
    "colorAvatarBackgroundPink": "#FF9788",
    "colorAvatarBackgroundYellow": "#FDE68A",
    "colorBackground": "#FFFFFF",
    "colorBackgroundBody": "#F7F9FF",
    "colorBackgroundBodyStrong": "#1E293B",
    "colorBackgroundComplete": "linear-gradient(360deg, rgba(82, 244, 174, 0.12) 0%, rgba(131, 247, 197, 0.0575) 57.81%, rgba(255, 255, 255, 0) 100%)",
    "colorBackgroundDecorativeStrong": "#C8F0EF",
    "colorBackgroundDecorativeWeak": "#FFF1DB",
    "colorBackgroundDestructive": "#DC2626",
    "colorBackgroundDestructiveStrong": "#B91C1C",
    "colorBackgroundError": "#FEF2F2",
    "colorBackgroundInfo": "#EBEDFF",
    "colorBackgroundLinkVisited": "#F5EEFA",
    "colorBackgroundOutlineHover": "#EBEDFF",
    "colorBackgroundPreview": "linear-gradient(360deg, rgba(71, 94, 105, 0.08) 6.19%, rgba(71, 94, 105, 0.0504167) 57.3%, rgba(255, 255, 255, 0) 93.81%)",
    "colorBackgroundPrimary": "#102EE9",
    "colorBackgroundPrimaryStrong": "#0B1F9C",
    "colorBackgroundPrimaryStrongest": "#041162",
    "colorBackgroundPrimaryWeak": "#4A63FC",
    "colorBackgroundPrimaryWeakest": "#7084FF",
    "colorBackgroundStrong": "#CBD5E1",
    "colorBackgroundStrongest": "#64748B",
    "colorBackgroundSuccess": "#ECFDF5",
    "colorBackgroundTodo": "linear-gradient(360deg, rgba(16, 37, 233, 0.08) 0%, rgba(36, 63, 235, 0.0733333) 49.48%, rgba(255, 255, 255, 0) 100%)",
    "colorBackgroundWarning": "#FFFBEB",
    "colorBackgroundWeak": "#F1F5F9",
    "colorBackgroundWeaker": "#F7F9FF",
    "colorBackgroundWeakest": "#F8FAFC",
    "colorBorder": "#64748B",
    "colorBorderError": "#DC2626",
    "colorBorderPrimary": "#102EE9",
    "colorBorderStrongest": "#0F172A",
    "colorBorderSuccess": "#059669",
    "colorBorderWarning": "#D97706",
    "colorBorderWeaker": "#CBD5E1",
    "colorBorderWeakest": "#E2E8F0",
    "colorButtonBackgroundPrimary": "#102EE9",
    "colorButtonBackgroundPrimaryHover": "#041162",
    "colorButtonBackgroundPrimaryOutlineFocus": "#102EE9",
    "colorButtonTextPrimary": "#FFFFFF",
    "colorFooterAvatarDecoration": "#102EE9",
    "colorFooterBackground": "#C8F0EF",
    "colorFooterSecondaryText": "#0F172A",
    "colorFooterText": "#041162",
    "colorHeaderBackground": "#0B1F9C",
    "colorHeaderDecoration": "#6EE7B7",
    "colorHeaderTextInverse": "#FFFFFF",
    "colorIconError": "#DC2626",
    "colorIconInfo": "#102EE9",
    "colorIconMapPrimary": "#0B1F9C",
    "colorIconMapSecondary": "#0B1F9C",
    "colorIconStrong": "#334155",
    "colorIconStronger": "#0F172A",
    "colorIconSuccess": "#059669",
    "colorIconWarning": "#D97706",
    "colorIconWeak": "#64748B",
    "colorIconWeaker": "#94A3B8",
    "colorMapBackgroundPrimary": "#102EE9",
    "colorMapBackgroundSecondary": "#102EE9",
    "colorPrimary": "#102EE9",
    "colorProgressBarBackgroundPrimary": "#FFFFFF",
    "colorProgressBarIndicatorPrimary": "#102EE9",
    "colorSecondary": "#76FAC1",
    "colorSidebarButtonBackground": "#FFFFFF",
    "colorSidebarButtonHover": "#F1F5F9",
    "colorSidebarButtonSelectedBackground": "#F1F5F9",
    "colorSidebarButtonSelectedText": "#0B1F9C",
    "colorSidebarButtonText": "#334155",
    "colorSidebarButtonTextHover": "#0B1F9C",
    "colorStepProgressBackground": "#4A63FC",
    "colorStepProgressText": "#FFFFFF",
    "colorStepProgressTitle": "#FFFFFF",
    "colorText": "#64748B",
    "colorTextError": "#B91C1C",
    "colorTextHeading": "#334155",
    "colorTextHeadingStrong": "#102EE9",
    "colorTextHeadingStronger": "#041162",
    "colorTextInfo": "#0B1F9C",
    "colorTextInverse": "#FFFFFF",
    "colorTextLink": "#102EE9",
    "colorTextLinkStrong": "#0B1F9C",
    "colorTextLinkVisited": "#950FFF",
    "colorTextStronger": "#334155",
    "colorTextStrongest": "#0F172A",
    "colorTextSuccess": "#047857",
    "colorTextWarning": "#B45309",
    "contentError": "#B91C1C",
    "contentInfo": "#102EE9",
    "contentInverse": "#FFFFFF",
    "contentPrimary": "#0F1A2A",
    "contentSecondary": "#4F5865",
    "contentSuccess": "#047857",
    "contentTertiary": "#6C7684",
    "contentWarning": "#B05C09",
    "transparent": "transparent",
  },
  "durations": Object {
    "fast-in": "250ms",
    "fast-out": "200ms",
    "instant": "100ms",
    "slow-in": "300ms",
    "slow-out": "250ms",
  },
  "fontSizes": Object {
    "fontSize10": "0.75rem",
    "fontSize20": "0.875rem",
    "fontSize30": "1rem",
    "fontSize40": "1.25rem",
    "fontSize50": "1.5rem",
    "fontSize60": "2rem",
    "fontSize70": "2.5rem",
    "fontSize80": "3rem",
    "fontSize90": "3.375rem",
    "titleBody": "1rem",
    "titleDisplay": "2.5rem",
    "titleGroup": "1.125rem",
    "titleScreen": "2rem",
    "titleSection": "1.5rem",
    "titleSubsection": "1.25rem",
  },
  "fontWeights": Object {
    "fontWeightBold": "700",
    "fontWeightLight": "300",
    "fontWeightMedium": "500",
    "fontWeightRegular": "400",
  },
  "fonts": Object {
    "fontFamilyNotoSans": "'Noto Sans', sans-serif",
  },
  "gridTemplateColumns": Object {
    "1": "repeat(1, minmax(0, 1fr))",
    "10": "repeat(10, minmax(0, 1fr))",
    "11": "repeat(11, minmax(0, 1fr))",
    "12": "repeat(12, minmax(0, 1fr))",
    "2": "repeat(2, minmax(0, 1fr))",
    "3": "repeat(3, minmax(0, 1fr))",
    "4": "repeat(4, minmax(0, 1fr))",
    "5": "repeat(5, minmax(0, 1fr))",
    "6": "repeat(6, minmax(0, 1fr))",
    "7": "repeat(7, minmax(0, 1fr))",
    "8": "repeat(8, minmax(0, 1fr))",
    "9": "repeat(9, minmax(0, 1fr))",
  },
  "gridTemplateRows": Object {
    "1": "repeat(1, minmax(0, 1fr))",
    "2": "repeat(2, minmax(0, 1fr))",
    "3": "repeat(3, minmax(0, 1fr))",
    "4": "repeat(4, minmax(0, 1fr))",
    "5": "repeat(5, minmax(0, 1fr))",
    "6": "repeat(6, minmax(0, 1fr))",
  },
  "letterSpacings": Object {
    "normal": "0em",
    "tight": "-0.025em",
    "tighter": "-0.05em",
    "wide": "0.025em",
    "wider": "0.05em",
    "widest": "0.1em",
  },
  "lineHeights": Object {
    "lh10": "2.5rem",
    "lh12": "3rem",
    "lh4": "1rem",
    "lh4_5": "1.125rem",
    "lh5": "1.25rem",
    "lh6": "1.5rem",
    "lh7": "1.75rem",
    "lh8": "2rem",
    "lineHeight10": "1rem",
    "lineHeight100": "3.75rem",
    "lineHeight20": "1.125rem",
    "lineHeight30": "1.25rem",
    "lineHeight40": "1.5rem",
    "lineHeight50": "1.75rem",
    "lineHeight60": "2rem",
    "lineHeight70": "2.5rem",
    "lineHeight80": "3rem",
    "lineHeight90": "3.375rem",
  },
  "radii": Object {
    "borderRadius0": "0px",
    "borderRadius10": "4px",
    "borderRadius20": "6px",
    "borderRadius30": "8px",
    "borderRadius35": "12px",
    "borderRadius40": "16px",
    "borderRadius50": "24px",
    "borderRadiusCircle": "50%",
    "borderRadiusPill": "100px",
  },
  "ringWidths": Object {
    "default": 3,
  },
  "screens": Object {
    "2xl": 1536,
    "_": 0,
    "lg": 1024,
    "md": 768,
    "sm": 640,
    "xl": 1280,
    "xs": 0,
  },
  "shadows": Object {
    "none": "none",
    "shadow": "0px 3px 24px rgba(15, 23, 42, 0.05)",
    "shadowFocus": "0px 1px 2px rgba(0, 0, 0, 0.05), 0px 0px 0px 2px #FFFFFF, 0px 0px 0px 4px #102EE9",
    "shadowStrong": "0px 8px 24px rgba(15, 23, 42, 0.08)",
  },
  "sizes": Object {
    "sizeIcon10": "0.75rem",
    "sizeIcon100": "3rem",
    "sizeIcon110": "3.25rem",
    "sizeIcon120": "3.5rem",
    "sizeIcon130": "3.75rem",
    "sizeIcon140": "4rem",
    "sizeIcon150": "4.25rem",
    "sizeIcon160": "4.5rem",
    "sizeIcon20": "1rem",
    "sizeIcon30": "1.25rem",
    "sizeIcon40": "1.5rem",
    "sizeIcon50": "1.75rem",
    "sizeIcon60": "2rem",
    "sizeIcon70": "2.25rem",
    "sizeIcon80": "2.5rem",
    "sizeIcon90": "2.75rem",
  },
  "space": Object {
    "space0": "0rem",
    "space10": "0.125rem",
    "space100": "2.25rem",
    "space110": "2.5rem",
    "space120": "3rem",
    "space130": "3.25rem",
    "space140": "3.5rem",
    "space150": "3.75rem",
    "space160": "4.5rem",
    "space170": "5rem",
    "space180": "6rem",
    "space190": "6.25rem",
    "space20": "0.25rem",
    "space25": "0.4rem",
    "space30": "0.5rem",
    "space40": "0.75rem",
    "space50": "1rem",
    "space60": "1.25rem",
    "space70": "1.5rem",
    "space80": "1.75rem",
    "space90": "2rem",
    "spaceNegative10": "-0.125rem",
    "spaceNegative100": "-2.25rem",
    "spaceNegative110": "-2.5rem",
    "spaceNegative120": "-3rem",
    "spaceNegative130": "-3.25rem",
    "spaceNegative140": "-3.5rem",
    "spaceNegative150": "-3.75rem",
    "spaceNegative160": "-4.5rem",
    "spaceNegative170": "-5rem",
    "spaceNegative180": "-6rem",
    "spaceNegative190": "-6.25rem",
    "spaceNegative20": "-0.25rem",
    "spaceNegative30": "-0.5rem",
    "spaceNegative40": "-0.75rem",
    "spaceNegative50": "-1rem",
    "spaceNegative60": "-1.25rem",
    "spaceNegative70": "-1.5rem",
    "spaceNegative80": "-1.75rem",
    "spaceNegative90": "-2rem",
  },
  "states": Object {
    "_": null,
    "active": "&:active",
    "activeItem": "&[data-active-item]",
    "backdrop": "&[data-backdrop]",
    "checked": "&:checked, &[aria-checked=\\"true\\"], &[data-state=\\"checked\\"]",
    "disabled": "&:disabled, &[aria-disabled=true]",
    "even": "&:nth-child(even)",
    "first": "&:first-child",
    "focus": "&:focus",
    "focusVisible": "&:focus-visible",
    "focusWithin": "&:focus-within",
    "hidden": "&[data-hidden=\\"true\\"]",
    "hover": "&:hover",
    "last": "&:last-child",
    "loading": "&[aria-busy=\\"true\\"]",
    "motionReduce": "@media (prefers-reduced-motion: reduce)",
    "motionSafe": "@media (prefers-reduced-motion: no-preference)",
    "odd": "&:nth-child(odd)",
    "placeholder": "&::placeholder",
    "selected": "&[aria-selected=\\"true\\"]",
    "visited": "&:visited",
  },
  "texts": Object {
    "2xl": Object {
      "fontSize": "2xl",
      "lineHeight": "2xl",
    },
    "3xl": Object {
      "fontSize": "3xl",
      "lineHeight": "3xl",
    },
    "4xl": Object {
      "fontSize": "4xl",
      "lineHeight": "4xl",
    },
    "5xl": Object {
      "fontSize": "5xl",
      "lineHeight": "5xl",
    },
    "6xl": Object {
      "fontSize": "6xl",
      "lineHeight": "6xl",
    },
    "7xl": Object {
      "fontSize": "7xl",
      "lineHeight": "7xl",
    },
    "8xl": Object {
      "fontSize": "8xl",
      "lineHeight": "8xl",
    },
    "9xl": Object {
      "fontSize": "9xl",
      "lineHeight": "9xl",
    },
    "default": Object {
      "fontSize": "default",
      "lineHeight": "default",
    },
    "lg": Object {
      "fontSize": "lg",
      "lineHeight": "lg",
    },
    "sm": Object {
      "fontSize": "sm",
      "lineHeight": "sm",
    },
    "xl": Object {
      "fontSize": "xl",
      "lineHeight": "xl",
    },
    "xs": Object {
      "fontSize": "xs",
      "lineHeight": "xs",
    },
  },
  "timingFunctions": Object {
    "ease-in": "cubic-bezier(0.4, 0, 1, 1)",
    "ease-in-out": "cubic-bezier(0.4, 0, 0.2, 1)",
    "ease-out": "cubic-bezier(0, 0, 0.2, 1)",
  },
  "transitionProperties": Object {
    "colors": Array [
      "background-color",
      "border-color",
      "color",
      "fill",
      "stroke",
    ],
    "default": Array [
      "background-color",
      "border-color",
      "color",
      "fill",
      "stroke",
      "opacity",
      "box-shadow",
      "transform",
    ],
    "opacity": Array [
      "opacity",
    ],
    "shadow": Array [
      "box-shadow",
    ],
    "transform": Array [
      "transform",
    ],
  },
  "transitions": Object {
    "colors": "background-color cubic-bezier(0.4, 0, 0.2, 1) 150ms,border-color cubic-bezier(0.4, 0, 0.2, 1) 150ms,color cubic-bezier(0.4, 0, 0.2, 1) 150ms,fill cubic-bezier(0.4, 0, 0.2, 1) 150ms,stroke cubic-bezier(0.4, 0, 0.2, 1) 150ms",
    "default": "background-color cubic-bezier(0.4, 0, 0.2, 1) 150ms,border-color cubic-bezier(0.4, 0, 0.2, 1) 150ms,color cubic-bezier(0.4, 0, 0.2, 1) 150ms,fill cubic-bezier(0.4, 0, 0.2, 1) 150ms,stroke cubic-bezier(0.4, 0, 0.2, 1) 150ms,opacity cubic-bezier(0.4, 0, 0.2, 1) 150ms,box-shadow cubic-bezier(0.4, 0, 0.2, 1) 150ms,transform cubic-bezier(0.4, 0, 0.2, 1) 150ms",
    "opacity": "opacity cubic-bezier(0.4, 0, 0.2, 1) 150ms",
    "shadow": "box-shadow cubic-bezier(0.4, 0, 0.2, 1) 150ms",
    "transform": "transform cubic-bezier(0.4, 0, 0.2, 1) 150ms",
  },
  "xstyled": Object {
    "cache": true,
  },
  "zIndices": Object {
    "zIndex0": 0,
    "zIndex10": 10,
    "zIndex20": 20,
    "zIndex30": 30,
    "zIndex40": 40,
    "zIndex50": 50,
    "zIndex60": 60,
    "zIndex70": 70,
    "zIndex80": 80,
    "zIndex90": 90,
    "zIndexNegative1": -1,
  },
}
`;
